spylib/exceptions.py

```diff
@@ -27,7 +27,7 @@ class ShopifyExceedingMaxCostError(ShopifyError):
     pass
 
 
-def not_our_fault(exception: Exception):
+def not_our_fault(exception: BaseException):
```

Comment:
We don't want to catch [system-exiting exceptions](https://docs.python.org/3/library/exceptions.html) so let's use `Exception`
```suggestion
def not_our_fault(exception: Exception):
```

Reply:
Weird I was getting a type error yesterday because of this but now it's gone... Fixed on new commit